src/index.ts

```diff
@@ -1001,3 +1001,65 @@ export class Values extends EventEmitter {
     return values;
   }
 }
+
+export type TokenState = {
+  token: PosNegToken;
+  clickSubmitted: boolean;
+};
+
+export const POS_NEG_STORAGE_KEY = "sajari_tokens";
+
+/**
+ * PosNegLocalStorageManager is a static utility class for manipulating Sajari's localstorage
+ * based management of PosNeg tokens. Typical use case is storing tokens for later consumption
+ * as users move through an ecommerce purcahse funnel.
+ */
+export class PosNegLocalStorageManager {
```

Comment:
Do you think we need tests for this class?

Reply:
Yeah, that's fair.